### DISCUSSION TITLE
How to exclude a file from `serveStatic`

### DISCUSSION THREAD

**Discussion by charnould** [honojs/hono#3586] [Q&A] [answered]
Hello,

Again, thanks for making `Hono`!

Currently, I use (with `Bun`) `app.use('/assets/*', serveStatic({ root: './' }))` to serve static content.
And it works perfectly!

However in my `assets` folder I have a file (let's say `config.ts`) I do NOT want to serve. 
Do you know a way to exclude it? (I've tried with a Regex but without success).

Thanks and have a good day!

**Comment by maou-shonen** [comment, marked as answer]
This is in the official [docs](https://hono.dev/docs/getting-started/bun#serve-static-files),
for your example

```
const app = new Hono()

app.get('/assets/config.ts', (c) => c.notFound())
app.use('/assets/*', serveStatic({ root: './' }))
```

**Reply by charnould** [reply]
I've missed it (didn't get it). 
Thanks!
